docs/pages/components/_meta.json

```diff
@@ -0,0 +1,8 @@
+{
+  "index": "Installation",
```

Comment:
Title in the browser's navbar display undefined for the following page:
- https://docs-git-feature-documentation-improvements-liteflow.vercel.app/components
- https://docs-git-feature-documentation-improvements-liteflow.vercel.app/templates

Reply:
done, had to use a "markdown component" instead of the symlink directly so it doesn't break the readme but can still "improve" it